ccm_web/client/src/pages/GradebookCanvas.tsx

```diff
@@ -341,7 +366,8 @@ function ConvertCanvasGradebook (): JSX.Element {
       case GradebookCanvasPageState.InvalidUpload:
         return renderInvalidUpload()
       case GradebookCanvasPageState.Confirm:
-        return renderConfirm(gradeBookRecordToStudentGrade(pageState.grades))
+        // This is kind of an uncomfortable solution, but we want just a single warning rather than one for each row
```

Comment:
This comment makes somebody feel like this is hack or workaround. But in reality it is not. That is the way we wanted, I feel we can remove the comment from here

Reply:
Ok I removed it.  I left the comment there b/c it did feel hacky since it didn't fit within the design of the original requirements and was integrated in a way I would not have done it otherwise.